user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_event_config.md

```diff
@@ -146,6 +146,39 @@ Examples:
 
   ![Set value - time range](~/user-guide/images/Set_Value_Time_Range.gif)<br>*Time range, button, and line & area chart components in DataMiner 10.4.11*
 
+## Changing a variable
+
+Available from DataMiner 10.4.0 [CU10]/10.5.1 onwards<!--RN 41324 + 41253-->. This option is only displayed if at least one [variable](xref:Variables) is configured in the low-code app.
+
+To configure an event to change a variable:
+
+1. Select *Change variable*.
+
+1. In the *Variable* box, select the variable you want to modify.
+
+1. Depending on the type of variable, specify the action to be performed when the event is triggered:
+
+   - *Set [variable type]*<!--RN 41253-->: Set the value of the variable. Depending on the variable type, you can:
+
+     - Enter a static value.
+
+     - Select a value from the dropdown list.
+
+     - Link the value to dynamic data in the app using the ![Link to data](~/user-guide/images/Link_to_Data.png) icon.
+
+     > [!NOTE]
+     > For table variables, you can only set a static value. The columns in the table are locked to the default values specified during the variable setup.
+
+   - *Update* > *Add row*<!--RN 41324-->: Add a new row to a table variable. Enter the data for each column in the row.
```

Comment:
these values can be linked to any available data in the application, might be handy to mention it so users know.